On z/OS running Java 11 a FFDC with caused by AttachNotSupportedException occurs when feature localConnector-1.0  is specified.

**Describe the bug** 
When running on z/OS with Java 11, the following FFDC  is seen when starting a server that specifies the localConnector-1.0 feature.

>------Start of DE processing------ = [8/17/22, 20:31:49:741 GMT]
>Exception = java.lang.reflect.InvocationTargetException
>Source = com.ibm.ws.jmx.connector.local.LocalConnectorActivator$LocalConnectorHelper$1
>probeid = 139
>Stack Dump = java.lang.reflect.InvocationTargetException
>	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
>	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
>	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
>	at com.ibm.ws.jmx.connector.local.LocalConnectorActivator$LocalConnectorHelper$1.run(LocalConnectorActivator.java:125)
>	at com.ibm.ws.jmx.connector.local.LocalConnectorActivator$LocalConnectorHelper$1.run(LocalConnectorActivator.java:60)
>	at java.base/java.security.AccessController.doPrivileged(AccessController.java:682)
>	at com.ibm.ws.jmx.connector.local.LocalConnectorActivator$LocalConnectorHelper.initConnectorAddress(LocalConnectorActivator.java:60)
>	at com.ibm.ws.jmx.connector.local.LocalConnectorActivator$LocalConnectorHelper.<clinit>(LocalConnectorActivator.java:50)
>	at com.ibm.ws.jmx.connector.local.LocalConnectorActivator.createJmxAddressResource(LocalConnectorActivator.java:212)
>	at com.ibm.ws.jmx.connector.local.LocalConnectorActivator.createJMXWorkAreaResource(LocalConnectorActivator.java:204)
>	at com.ibm.ws.jmx.connector.local.LocalConnectorActivator.<init>(LocalConnectorActivator.java:161)
>	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
>	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
>	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
>	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
>	at org.apache.felix.scr.impl.inject.internal.ComponentConstructorImpl.newInstance(ComponentConstructorImpl.java:316)
>	at org.apache.felix.scr.impl.manager.SingleComponentManager.createImplementationObject(SingleComponentManager.java:286)
>	at org.apache.felix.scr.impl.manager.SingleComponentManager.createComponent(SingleComponentManager.java:115)
>	at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:1000)
>	at org.apache.felix.scr.impl.manager.SingleComponentManager.getServiceInternal(SingleComponentManager.java:973)
>	at org.apache.felix.scr.impl.manager.AbstractComponentManager.activateInternal(AbstractComponentManager.java:785)
>	at org.apache.felix.scr.impl.manager.DependencyManager$SingleStaticCustomizer.addedService(DependencyManager.java:1271)
>	at org.apache.felix.scr.impl.manager.DependencyManager$SingleStaticCustomizer.addedService(DependencyManager.java:1222)
>	at org.apache.felix.scr.impl.manager.ServiceTracker$Tracked.customizerAdded(ServiceTracker.java:1200)
>	at org.apache.felix.scr.impl.manager.ServiceTracker$Tracked.customizerAdded(ServiceTracker.java:1121)
>	at org.apache.felix.scr.impl.manager.ServiceTracker$AbstractTracked.trackAdding(ServiceTracker.java:928)
>	at org.apache.felix.scr.impl.manager.ServiceTracker$AbstractTracked.track(ServiceTracker.java:864)
>	at org.apache.felix.scr.impl.manager.ServiceTracker$Tracked.serviceChanged(ServiceTracker.java:1152)
>	at org.apache.felix.scr.impl.BundleComponentActivator$ListenerInfo.serviceChanged(BundleComponentActivator.java:114)
>	at org.eclipse.osgi.internal.serviceregistry.FilteredServiceListener.serviceChanged(FilteredServiceListener.java:120)
>	at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:956)
>	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
>	at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:151)
>	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.publishServiceEventPrivileged(ServiceRegistry.java:936)
>	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.publishServiceEvent(ServiceRegistry.java:873)
>	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistrationImpl.register(ServiceRegistrationImpl.java:141)
>	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.registerService(ServiceRegistry.java:261)
>	at org.eclipse.osgi.internal.framework.BundleContextImpl.registerService(BundleContextImpl.java:495)
>	at org.eclipse.osgi.internal.framework.BundleContextImpl.registerService(BundleContextImpl.java:514)
>	at org.eclipse.osgi.internal.framework.BundleContextImpl.registerService(BundleContextImpl.java:1042)
>	at com.ibm.ws.kernel.feature.internal.FeatureManager.update(FeatureManager.java:791)
>	at com.ibm.ws.kernel.feature.internal.FeatureManager.processFeatureChanges(FeatureManager.java:887)
>	at com.ibm.ws.kernel.feature.internal.FeatureManager$1.run(FeatureManager.java:673)
>	at com.ibm.ws.threading.internal.ExecutorServiceImpl$RunnableWrapper.run(ExecutorServiceImpl.java:245)
>	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
>	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
>	at java.base/java.lang.Thread.run(Thread.java:871)
>Caused by: com.sun.tools.attach.AttachNotSupportedException: target 16843614 not found
>	at com.ibm.tools.attach.attacher.OpenJ9VirtualMachine.attachTargetImpl(OpenJ9VirtualMachine.java:150)
>	at com.ibm.tools.attach.attacher.OpenJ9VirtualMachine.lambda$attachTarget$1(OpenJ9VirtualMachine.java:128)
>	at com.ibm.tools.attach.attacher.OpenJ9VirtualMachine$$Lambda$214/0x0000000000000000.run(Unknown Source)
>	at java.base/java.security.AccessController.doPrivileged(AccessController.java:738)
>	at com.ibm.tools.attach.attacher.OpenJ9VirtualMachine.attachTarget(OpenJ9VirtualMachine.java:130)
>	at com.ibm.tools.attach.attacher.OpenJ9AttachProvider.attachVirtualMachine(OpenJ9AttachProvider.java:64)
>	at com.ibm.tools.attach.attacher.OpenJ9AttachProvider.attachVirtualMachine(OpenJ9AttachProvider.java:47)
>	at com.sun.tools.attach.VirtualMachine.attach(VirtualMachine.java:207)
]
	at componenttest.custom.junit.runner.FATRunner.blowup(FATRunner.java:329)
	at componenttest.custom.junit.runner.FATRunner.access$600(FATRunner.java:66)
	at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:279)
	at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:347)
	at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:171)

**Steps to Reproduce**
Add the localConnector-1.0 feature to server.xml and run on z/OS with Java 11.

**Expected behavior** 
Server should start without getting a FFDC.

--- a/dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server
+++ b/dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server
@@ -713,7 +713,12 @@ serverEnvAndJVMOptions()
 
   # Avoids HeadlessException on all platforms and Liberty JVMs appearing as applications and stealing focus on Mac.
   # Declare allowAttachSelf=true so that the VM will permit a late self attach if localConnector-1.0 is enabled
-  JVM_OPTIONS_QUOTED="-Djava.awt.headless=true -Djdk.attach.allowAttachSelf=true"
+  # need -Dcom.ibm.tools.attach.enable=yes for java 11 on z/OS
+  if [ "${uname}" = OS/390 ]; then
+    JVM_OPTIONS_QUOTED="-Djava.awt.headless=true -Djdk.attach.allowAttachSelf=true -Dcom.ibm.tools.attach.enable=yes"
+  else
+    JVM_OPTIONS_QUOTED="-Djava.awt.headless=true -Djdk.attach.allowAttachSelf=true"
+  fi
   SERVER_JVM_OPTIONS_QUOTED=${JVM_OPTIONS_QUOTED}
 
   rc=0

